content/sensu-go/6.6/release-notes.md

```diff
@@ -99,6 +100,30 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.6.4 release notes
+
+**January 13, 2022** &mdash; The latest release of Sensu Go, version 6.6.4, is now available for download.
+
+Sensu Go 6.6.4 includes a number of bug fixes, security improvements, and a new metric, `sensu_go_event_metric_points_processed`.
+Fixes in this patch release will help prevent backend crashes when PostgreSQL is taken offline and keep backend entity rows from filling up the entities table.
+The 6.6.4 patch release also includes several improvements to further secure the web UI.
+
+Read the [upgrade guide][1] to upgrade Sensu to version 6.6.4.
+
+**IMPROVEMENTS**
+- ([Commercial feature][259]) In the web UI, added the [X-Frame-Options][263] header to tell browsers the web application cannot be loaded within an iframe to prevent tailored click-jacking attacks. 
+- ([Commercial feature][259]) In the web UI, added [HSTS header][264] if TLS has been configured; ensures the browser loads the application and its requisite assets with a secure connection.
+- ([Commercial feature][259]) In the web UI, added [X-Content-Type-Options][265]: nosniff header to have browsers respect the given Content-Type header when loading content referenced by a script tag. 
+- Add sensu_go_event_metric_points_processed counter metric and included it in tessen reporting.
+
+**FIXES**
+- ([Commercial feature][259]) Fixed bugs in BSM and round robin scheduling. When postgres round robin scheduling is enabled, users will no longer see missed check executions.
```

Comment:
```suggestion
- ([Commercial feature][259]) Fixed bugs in business service monitoring (BSM) and round robin scheduling to prevent missed check executions when PostgreSQL round robin scheduling is enabled.
```